lms/templates/seq_module.html

```diff
@@ -44,7 +44,7 @@
      ${item['content'] | h}
   </div>
   % endfor
-  <div id="seq_content" role="tabpanel"></div>
+  <div id="seq_content" tabindex="-1"></div>
 
   <nav class="sequence-bottom">
     <ul aria-label="${_('Section Navigation')}" class="sequence-nav-buttons">
```

Comment:
this aria-label needs to go on the nav element 1 line up.  The word "Navigation" is not necessary as the SR will say that when it announces the type of landmark.